test/numbers.jl

```diff
@@ -2645,10 +2645,10 @@ let T = Rational
     yi = 4
 
     @test x^y ≈ big(xf)^big(yf)
-    @test x^yi ≈ big(xf)^yi
-    @test x^true ≈ big(xf)^true
-    @test x^false ≈ big(xf)^false
-    @test x^1 ≈ big(xf)^1
+    @test_broken x^yi ≈ big(xf)^yi
+    @test_broken x^true ≈ big(xf)^true
+    @test x^false == big(xf)^false
+    @test_broken x^1 ≈ big(xf)^1
```

Comment:
Because these are not close enough for `≈`. The `@test_approx_eq` macro was hiding this with it's massive brokenness on unusual numeric types, including `Float16` and `BigFloat`. I don't have the expertise in this area to know how this should be fixed. At least now we know it's broken.